### DISCUSSION TITLE
What are the differences between a Nuxt 2, Nuxt 2 + Bridge, and Nuxt 3 module?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [nuxt/nuxt#17510] [Questions]
I'm curious what the differences are between Nuxt 2, Nuxt 2 + Bridge, and Nuxt 3 modules? I'm finding the biggest issue in migrating a Nuxt 2 app is the module dependencies. I'd like to update some of these modules to work with modern Nuxt.

**Comment by pi0** [comment]
Hi! Here is a quick comparison table: https://v3.nuxtjs.org/getting-started/migration

Bridge is the best if you have dependencies on Vue 2 and brings as much as possible goodies to your current app. Nuxt 3 is the way to go otherwise!
